packages/component-library/draft/Kaizen/HeroCard/README.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: HeroCard
+navTitle: HeroCard
```

Comment:
These can be "Hero Card" instead of "HeroCard" (no need for camel casing, they can be separate words).